commerce_coordinator/apps/commercetools/utils.py

```diff
@@ -130,18 +139,41 @@ def extract_ct_product_information_for_braze_canvas(item: LineItem):
     return result
 
 
+def calculate_discount(order: Order) -> TypedMoney:
```

Comment:
I suggest this name or sth similar
```suggestion
def calculate_total_discount_on_order(order: Order) -> TypedMoney:
```